src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -177,8 +221,18 @@ public void Save(ModelSaveContext ctx)
             ctx.SetVersionInfo(GetVersionInfo());
 
             ctx.SaveBinaryStream("OnnxModel", w => { w.WriteByteArray(Model.ToByteArray()); });
-            ctx.SaveNonEmptyString(_args.InputColumn);
-            ctx.SaveNonEmptyString(_args.OutputColumn);
+            //ctx.SaveNonEmptyString(_args.InputColumn);
+            //ctx.SaveNonEmptyString(_args.OutputColumn);
```

Comment:
delete

Reply:
done

---
In reply to: [232837354](https://github.com/dotnet/machinelearning/pull/1586#discussion_r232837354) [](ancestors = 232837354)